gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStatsJdbcMonitor.java

```diff
@@ -62,7 +62,7 @@ public ClusterStats monitor(ProxyBackendConfiguration backend) {
       jdbcUrl = String
```

Comment:
Since we are getting smarter about setting the port based on the protocol, we should also use the backend URL to set the SSL parameter. Currently it is set in `BackendStateConfiguration`, so if you have a mixture of `http` and `https` backends it will necessarily be invalid for some of them.

Reply:
I'm not familiar with SSL parameters. So I'm assuming SSL can be set depending on http or https?

Reply:
yep  
https => SSL=true 
http => SSL=false

Reply:
Lets add that and then I think we should be good to merge this soon.